lib/screen/customer_support/support_thread_page.dart

```diff
@@ -146,6 +184,12 @@ class _SupportThreadPageState extends State<SupportThreadPage> {
         createdAt: DateTime.now().millisecondsSinceEpoch,
       );
 
+  void _setIssueId(String issueId) {
```

Comment:
This function not only set issueID but also update the _announcement, I think we should use `onIssueIDChanged` instead.

Reply:
this function is to set the _issueId. As the issueId is change, the announce also change to.